_data/simple-icons.json

```diff
@@ -9239,6 +9239,11 @@
             "hex": "ED1C24",
             "source": "https://tinyletter.com/site/press"
         },
+        {
+            "title": "Tippy.js",
+            "hex": "213B50",
+            "source": "https://atomiks.github.io/tippyjs/static/logo-ebc385458e03fdb24af078536af88065.svg"
```

Comment:
Did not have time to review the icon yet, but I found the source in their Github repo. Can you use this link instead as source:
https://github.com/atomiks/tippyjs/blob/8f7b2df156ae39c4df23c481b77ab59fe7f08c44/website/src/images/logo.svg